README.md

```diff
@@ -156,222 +145,144 @@ pluginsync=true
 
+3. Configure reports in the `[main]` section of `puppet.conf`:
+    ```conf
+    [main]
+    reports=datadog_reports
+    ```
+4. Run puppet in masterless configuration:
+    ```shell
+    puppet apply --modulepath <path_to_modules> <path_to_site.pp>
+    ```
 
-Client Settings
-===============
+## Client settings
 
 ### Tagging client nodes
 
 The Datadog Agent configuration file is recreated from the template every Puppet run. If you need to tag your nodes, add an array entry in Hiera:
 
-```
+```conf
 datadog_agent::tags:
 - 'keyname:value'
 - 'anotherkey:%{factname}'
 ```
 
+### Extra configuration variables
+
 These variables can be set in the `datadog_agent` class to control settings in the Agent:
 
 | variable name               | description                                                                                                                                                                                      |
 |-----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `agent_major_version`       | The version of the Agent to install: either 5, 6 or 7 (default: 7).                                                                                                                              |
-| `agent_version`             | Lets you pin a specific minor version of the agent to install, eg: `1:7.16.0-1`. Leave empty to install the latest version (not recommended).                                                    |
-| `collect_ec2_tags`          | Set this to yes to have an instance's custom EC2 tags used as agent tags.                                                                                                                        |
-| `collect_instance_metadata` | Set this to yes to have an instance's EC2 metadata used as agent tags.                                                                                                                           |
-| `datadog_site`              | The Datadog site to report to. Defaults to `datadoghq.com`, set to `datadoghq.eu` to report to the EU site (Agent versions 6 and 7 only).                                                        |
+| `agent_version`             | Lets you pin a specific minor version of the Agent to install, for example: `1:7.16.0-1`. Leave empty to install the latest version.                                                             |
+| `collect_ec2_tags`          | Collect an instance's custom EC2 tags as Agent tags by using `yes`.                                                                                                                              |
+| `collect_instance_metadata` | Collect an instance's EC2 metadata as Agent tags by using `yes`.                                                                                                                                 |
```

Comment:
These are `true`/`false` booleans now (for a few years actually 😅), instead of `yes`/`no`.